deployment/refinery-modules/refinery/manifests/apache2.pp

```diff
@@ -40,6 +40,11 @@
     content => "AcceptFilter http none\nAcceptFilter https none",
   }
 
+  # log Django messages only
+  apache::custom_config { 'error-log-format':
+    content => 'ErrorLogFormat "%M"',
+  }
+
```

Comment:
*Only* the error messages? Time and perhaps some request info could be useful...

Reply:
Django error messages contain timestamps. Request info goes into the access log.